### PR TITLE
Updated search bar design

### PR DESCRIPTION
**Pull request by spoorthyv**
Originally PR #500 

Updated search bar design with placeholder text and blue search icon on the side.

This is how is looks now:
screen shot 2016-08-16 at 10 34 44 am

@jasongrout @blink1073 @afshin @sccolbert

### CODE REVIEW

File: src/commandpalette/plugin.ts [hunk 1]
```diff
@@ -131,5 +131,7 @@ function activateCommandPalette(app: JupyterLab): ICommandPalette {
 
   app.shell.addToLeftArea(palette);
 
-  return new Palette(palette);
+  let palette2 = new Palette(palette);
```
Comment (blink1073) [inline src/commandpalette/plugin.ts]: `placeholder` is already an attribute of the original `palette`.

**Comment by spoorthyv** [comment]
Hmm. If I change the last couple lines to:

```
palette.placeholder = "SEARCH";
app.shell.addToLeftArea(palette);
return palette; 
```


It says  "Property 'placeholder' does not exist on type 'CommandPalette'." and "Type 'CommandPalette' is not assignable to type 'ICommandPalette'" in terminal.

**Comment by blink1073** [comment]
Right, sorry, it should be `palette.inputNode.placeholder`:

https://github.com/jupyter/jupyterlab/blob/master/src/commandpalette/plugin.ts#L40

**Comment by spoorthyv** [comment]
Ok. Should be fixed!

File: src/default-theme/commandpalette.css [hunk 2]
```diff
@@ -8,24 +8,26 @@
...
+  padding: 14px 12px;
   background: white;
-  border: 1px solid #E0E0E0;
+  border-bottom: 2px solid #E0E0E0;
+  overflow: overlay;
 }
 
+.p-CommandPalette-wrapper::after {
+    font-family: FontAwesome;
+    content: "\f002";
```
Comment (blink1073) [inline src/default-theme/commandpalette.css]: Can you add an inline comment here about which icon this is?

**Comment by blink1073** [comment]
LGTM, thank you!

**Pull request merged**